src/domain/collaboration/callout/CalloutsPreview/InnovationFlowCalloutsPreview.tsx

```
@@ -0,0 +1,125 @@
+        callout =>
+          selectedState &&
+          callout.framing.profile.flowStateTagset?.tags &&
+          callout.framing.profile.flowStateTagset?.tags.includes(selectedState)
+      )
+      .sort((a, b) => a.sortOrder - b.sortOrder);
+  }, [callouts, selectedState]);
+
+  const calloutDescriptions: Record<string, ReactNode> = useMemo(
+    () =>
+      visibleCallouts?.reduce(
+        (obj, callout) => ({ ...obj, [callout.id]: <CalloutDescription callout={callout} /> }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)